website/src/main/tut/docs/links.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: docs
+title: Links
+---
+
+# Links
+
+- May 3rd 2017: [Move fast and refactor with scalafix](https://vimeo.com/channels/flatmap2017/216469977)
```

Comment:
You should add your talk from Scala World ;)

Reply:
As soon as it goes live :)